backendTest.py

```diff
@@ -0,0 +1,24 @@
+import unittest

```

Comment:
There is a directory called `/test/unit`. Can you move these tests in there? Also, what is the `suite = unittest.TestLoader().loadTestsFromTestCase(BackEndTests)` stuff for?